### ISSUE TITLE
C API: example for using converters

### ISSUE THREAD

**Issue by paulflang** [sbmlteam/libsbml#143]
It seems that [example code](http://sbml.org/Software/libSBML/docs/c-api/class_conversion_properties__t.html) for using converters is missing in the docs for the C API:

![Screenshot 2021-05-03 145523](https://user-images.githubusercontent.com/47160416/116885317-aa88fc80-ac1f-11eb-9753-0e8c0a99b971.png)

**Comment by fbergmann** [comment]
Indeed, it seems that the example for documentation is only available for C++. However, we do have an example on how to use the converter here: 

<https://github.com/sbmlteam/libsbml/blob/development/examples/c/stripPackage.c>
